iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/iot/logdispatcher/LogDispatcher.java

```diff
@@ -344,7 +352,7 @@ public void updateSafelyDeletedSearchIndex() {
       // update safely deleted search index to delete outdated info,
       // indicating that insert nodes whose search index are before this value can be deleted
       // safely
```

Comment:
Add a few comments on why minFlushedIndex is used here

Reply:
Done.